lib/PubSub.jsx

```diff
@@ -1,7 +1,7 @@
-import _ from 'underscore';
 import has from 'lodash/has';
+import {once} from 'lodash';
```

Comment:
same here - aren't we removing lodash?

Reply:
The same [answer from above](https://github.com/Expensify/expensify-common/pull/704#discussion_r1636966712) applies to it